app.rb

```diff
@@ -0,0 +1,114 @@
+require_relative 'rental'
+require_relative 'person'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+
+class App
+  attr_accessor :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def start
+    puts 'Welcome to the Library System!'
+    list_of_options
+  end
+
+  def list_all_books
+    if @books.length.positive?
+      @books.each do |book|
+        puts "Title: #{book.title}, Author: #{book.author}"
+      end
+    else
+      puts 'No books found'
+    end
+  end
+
+  def list_all_people
+    @people.each do |person|
+      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]:'
+    person_type = gets.chomp
+    case person_type
+    when '1'
+      create_a_student
```

Comment:
- Hey, there seems to be a glitch in the app as you're calling the wrong name. you've used `create_student` instead of `create_a_student`. Kindly try testing your app by calling all the methods to see this error so you could fix them to improve the functionality of your app.

Reply:
### student.rb file


- There's also an error in the number of arguments and this is breaking your app currently, kindly test run your app to see all the errors so you could fix them as much as you can. See the image below:

![Screenshot from 2023-01-05 22-42-39](https://user-images.githubusercontent.com/59397159/210885898-217bc977-0ac1-49e8-a219-bfa16779bda4.png)

Reply:
Hii @zieeco, 

I didn't notice that, Thanks for the reminder. 😄

Reply:
This issue have been resolved. 😄